lib/ios-deploy.js

```diff
@@ -204,22 +205,62 @@ class IOSDeploy {
       }
       const executableName = apps[bundleId].CFBundleExecutable;
       log.debug(`The executable name for the bundle id '${bundleId}' was '${executableName}'`);
-      instrumentService = await services.startInstrumentService(this.udid);
-      const processes = await instrumentService.callChannel(
-        INSTRUMENT_CHANNEL.DEVICE_INFO,
-        'runningProcesses',
-      );
-      const process = processes.selector.find((process) => process.name === executableName);
-      if (!process) {
-        log.info(`The process of the bundle id '${bundleId}' was not running`);
-        return false;
+
+      // 'devicectl' has overhead than the instrument service via appium-ios-device,
+      // so hre uses the 'devicectl' only for iOS 17+.
+      if (util.compareVersions(platformVersion, '>', '17.0')) {
```

Comment:
>= ?

Reply:
Can we simply check if devicectl is available without verifying the platform version? It would be interesting to know if this would work with ios16

Reply:
the command itself has overhead rather than calling appium-ios-device's method for now, so I wanted to make the error minimal by the devicectl

Reply:
Interestingly, the `devicectl` did not identify iOS 16 device...

```
ERROR: The specified device was not found. (com.apple.dt.CoreDeviceError error 1000.)
```